react/src/views/Box/BoxView.tsx

```diff
@@ -248,27 +248,35 @@ function BTBox() {
 
   const onSubmitTakeItemsFromBox = (boxFormValues: IChangeNumberOfItemsBoxData) => {
     if (boxFormValues.numberOfItems && boxFormValues.numberOfItems > 0 && boxData?.numberOfItems) {
-      updateNumberOfItemsMutation({
-        variables: {
-          boxLabelIdentifier: labelIdentifier,
-          numberOfItems: (boxData?.numberOfItems || 0) - (boxFormValues?.numberOfItems || 0),
-        },
-      })
-        .then((res) => {
-          createToast({
-            title: `Box ${boxData.labelIdentifier}`,
-            type: res.errors ? "error" : "success",
-            message: res.errors
-              ? "Error: Could not remove items from the box"
-              : `Successfully removed ${boxFormValues?.numberOfItems} items from box`,
-          });
-          onMinusClose();
+      if (boxFormValues.numberOfItems > boxData?.numberOfItems) {
+        createToast({
+          title: `Box ${boxData.labelIdentifier}`,
+          type: "error",
+          message: `Could not remove more than ${boxData?.numberOfItems} items`,
+        });
+      } else {
+        updateNumberOfItemsMutation({
+          variables: {
+            boxLabelIdentifier: labelIdentifier,
+            numberOfItems: (boxData?.numberOfItems || 0) - (boxFormValues?.numberOfItems || 0),
```

Comment:
I think in line 250 it's already checked for values being defined, so the `|| 0` fallback might be redundant here? Or does eslint complain?

Reply:
@pylipp yes indeed eslint complain